sdks/python/apache_beam/runners/common.py

```diff
@@ -565,9 +588,14 @@ def process_outputs(self, windowed_input_element, results):
     then dispatched to the appropriate indexed output.
     """
     if results is None:
+      # TODO[BEAM-3937]:Remove if block after output feature released.
+      if self.per_element_output_counter:
+        self.per_element_output_counter.update(0)
       return
 
+    output_element_count = 0
     for result in results:
+      output_element_count += 1
```

Comment:
<!--new_thread; commit:c89749a0d77350dfbd2686db975aa087bafb8f1b; resolved:0-->
Please add comment that we use this increment pattern because `results` may be an iterable.